crud.py

```diff
@@ -0,0 +1,63 @@
+from sqlalchemy.orm import Session, Query
+
+import models
+import schemas
+
+
+def create_author(
+        db: Session,
+        author: schemas.AuthorCreate
+) -> models.DBAuthor:
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_authors_list(
+        db: Session,
+        skip: int = 0,
+        limit: int = 100
+) -> Query:

```

Comment:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/65166203/230024660-59ac5249-6a72-47cd-a7ef-4216d4f3cb14.png">
Is really return type Query? Fix in all places